isthmus/src/main/java/io/substrait/isthmus/expression/RexExpressionConverter.java

```diff
@@ -2,30 +2,45 @@
 
 import io.substrait.expression.Expression;
 import io.substrait.expression.FieldReference;
+import io.substrait.function.ImmutableSimpleExtension;
+import io.substrait.function.SimpleExtension;
 import io.substrait.isthmus.CallConverter;
+import io.substrait.isthmus.SubstraitRelVisitor;
 import io.substrait.isthmus.TypeConverter;
+import io.substrait.relation.Rel;
 import io.substrait.type.StringTypeVisitor;
+import java.io.IOException;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 import org.apache.calcite.rex.*;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
 
 public class RexExpressionConverter implements RexVisitor<Expression> {
   static final org.slf4j.Logger logger =
       org.slf4j.LoggerFactory.getLogger(RexExpressionConverter.class);
 
   private final List<CallConverter> callConverters;
+  private final SubstraitRelVisitor relVisitor;
+
+  public RexExpressionConverter(SubstraitRelVisitor relVisitor, CallConverter... callConverters) {
+    this(relVisitor, Arrays.asList(callConverters));
+  }
 
   public RexExpressionConverter(CallConverter... callConverters) {
-    this(Arrays.asList(callConverters));
+    this(null, Arrays.asList(callConverters));
   }
 
-  public RexExpressionConverter(List<CallConverter> callConverters) {
+  public RexExpressionConverter(
+      SubstraitRelVisitor relVisitor, List<CallConverter> callConverters) {
     this.callConverters = callConverters;
+    this.relVisitor = relVisitor;
   }
 
   public RexExpressionConverter() {
     this.callConverters = CallConverters.DEFAULTS;
+    this.relVisitor = null; // TODO
```

Comment:
What's the TODO here?

Reply:
This is for the case where RexExpressionConverter is used for Rex expression without nesting subquery.  I changed TODO to a comment.